lampod/src/actions/handler.rs

```diff
@@ -263,12 +264,13 @@ impl Handler for LampoHandler {
                 ..
             } => {
                 let (payment_preimage, payment_secret) = match purpose {
-                    ldk::events::PaymentPurpose::InvoicePayment {
+                    ldk::events::PaymentPurpose::Bolt11InvoicePayment {
                         payment_preimage,
                         payment_secret,
                         ..
                     } => (payment_preimage, Some(payment_secret)),
                     ldk::events::PaymentPurpose::SpontaneousPayment(preimage) => (Some(preimage), None),
+                    _ => panic!("BOLT 12 not implemented"),
```

Comment:
This is wrong, lampo support bolt12, so we should look inside what it is going on here